easybuild/easyconfigs/q/qmflows/qmflows-0.12.1-foss-2021a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'PythonBundle'
+
+name = 'qmflows'
+version = '0.12.1'
+
+homepage = 'https://github.com/SCM-NV/qmflows'
+description = """This library tackles the construction and efficient execution 
+of computational chemistry workflows. This allows computational chemists to use 
+the emerging massively parallel compute environments in an easy manner and focus 
+on interpretation of scientific data rather than on tedious job submission 
+procedures and manual data processing.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = {
+    ('Python', '3.9.5'),
+    ('RDKit', '2021.03.4'),
+    ('HDF5', '1.10.7'),
+    ('SciPy-bundle', '2021.05'),
+    ('Rust', '1.52.1')
+}
+
+use_pip = True
+
+exts_list = [
+    ('more-itertools', '9.0.0', {
+        'checksums': ['5a6257e40878ef0520b1803990e3e22303a41b5714006c32a3fd8304b26ea1ab'],
+    }),
+    ('dill', '0.3.6', {
+        'checksums': ['e5db55f3687856d8fbdab002ed78544e1c4559a130302693d839dfe8f93f2373'],
+    }),
+    ('plams', '1.5.1', {
+        'modulename': 'scm.plams',
+        'checksums': ['5e77632be84a8c12172eb426589cb80187b5b31325d917bdbeb0798f9c5aed46'],
+    }),
+    ('pyparsing', '3.0.9', {
+        'checksums': ['2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb'],
+    }),
+    ('PyYAML', '6.0', {
+        'modulename': 'yaml',
+        'checksums': ['68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2'],
+    }),
+    ('filelock', '3.8.0', {
+        'checksums': ['55447caa666f2198c5b6b13a26d2084d26fa5b115c00d065664b2124680c4edc'],
+    }),
+    ('packaging', '20.9', {
+        'checksums': ['5b327ac1320dc863dca72f4514ecc086f31186744b84a230374cc1fd776feae5'],
+    }),
+    ('graphviz', '0.19.2', {
+        'checksums': ['7c90cebc147c18bcdffcd3c76db58cbface5d45fe0247a2f3bfb144d32a8c77c'],
+        'sources': {'filename': '%(name)s-%(version)s.zip'}
+    }),
+    ('grpcio', '1.50.0', {
+        'modulename': 'grpc',
+        'checksums': ['12b479839a5e753580b5e6053571de14006157f2ef9b71f38c56dc9b23b95ad6'],
+    }),
+    ('protobuf', '4.21.9', {
```

Comment:
There is already an easyconfig with the python bindings of protobuf, this extension can be replaced by a regular dependency on `protobuf-python`.

Reply:
Agreed. Latest commit introduces dependency on https://github.com/easybuilders/easybuild-easyconfigs/pull/16937 and https://github.com/easybuilders/easybuild-easyconfigs/pull/16938.

Reply:
@guacke thanks for the update. Next issue is that you are adding a new version of protobuf to this toolchain generation (2021a), but our repository of easyconfigs is limited to a single version per dependency per toolchain. This is a policy established to avoid an explosion in the number of easyconfigs in the repo. You can use this combination of versions locally if it works, but we will have to reject it by policy.

Having said that, I fail to find the dependency on `protobuf`. Neither `qmflow` or its [dependencies](https://github.com/SCM-NV/qmflows/blob/cd03c1c8caa3ce8217515ea5ee632a532c13ea61/setup.py#L81) list protobuf as a requirement. Did I miss some information or did you add it for convenience? Maybe it can be removed altogether to simplify this easyconfig.